static/src/javascripts/projects/commercial/modules/sticky-mpu.js

```diff
@@ -19,12 +19,15 @@ const onResize = (specs, _, iframe: ?HTMLElement) => {
     }
 };
 
-const stickyCommentsMpu = (adSlot: HTMLElement) => {
-    const dataName = adSlot.getAttribute('data-name');
-    if (dataName !== 'comments') {
-        return;
+const stickyMpuDataName = (adSlot: HTMLElement) => {
+    const dataName = adSlot.dataset.name;
+    if (dataName !== 'comments' || dataName !== 'right') {
+        return false;
     }
+};
```

Comment:
Perhaps call this function `isStickyMpuSlot` and return 
`dataName === 'comments' || dataName === 'right'` 
?